js/w3c/headers.js

```diff
@@ -405,8 +405,8 @@ define(
                 conf.dashDate = utils.concatDate(conf.publishDate, "-");
                 conf.publishISODate = utils.isoDate(conf.publishDate);
                 conf.shortISODate = conf.publishISODate.replace(/T.*/, "");
-                conf.processVersion = conf.processVersion || "2014";
-                conf.isNewProcess = conf.processVersion == "2014";
+                conf.processVersion = conf.processVersion || "2015";
+                conf.isNewProcess = conf.processVersion == "2015";
```

Comment:
This is wrong. Any document that has its `processVersion` set to `2014` will, with the code above, suddenly **NOT** be under the new process. You need to check that it is under 2014 or 2015.